components/automate-ui/README.md

```diff
@@ -83,13 +83,16 @@ That is a guess on my part, could not really confirm what version one should be
 
 **Package diff2html: ^2.12.2**
 
```

Comment:
This section of the readme details the constrained packages; just a couple updates here.